test/e2e/set/scyllacluster/multi_datacenter_scyllacluster_external_seeds.go

```diff
@@ -0,0 +1,340 @@
+// Copyright (C) 2024 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+var _ = g.Describe("MultiDC cluster", framework.MultiDatacenter, func() {
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should form when external seeds are provided to ScyllaClusters", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), multiDatacenterTestTimeout)
+		defer cancel()
+
+		sc1 := f.GetDefaultScyllaCluster()
+		sc1.Name = "multi-datacenter-cluster"
+		sc1.Spec.Datacenter.Name = "dc1"
+		sc1.Spec.Sysctls = []string{"fs.aio-max-nr=30000000"}
+		sc1.Spec.Datacenter.Racks = []scyllav1.RackSpec{
+			{
+				Name:    "a",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "b",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "c",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+		}
+
+		framework.By("Ensuring default namespace presence in the first cluster")
```

Comment:
how? this is a getter only afaik

Reply:
The sentence, as it's phrased at this point, suggests it would be created if missing.

Reply:
It's also not really making an "external" action and more testing the framework, maybe not worthy of the `By`

Reply:
removed